GUITests/Tests/SignInTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using DTO.RestRequests;
+using GUITests.Tests.TestUtils;
+using GUITestsEngine.Utils.Attributes;
+using Kernel;
+using Kernel.Enums;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+
+namespace GUITestsEngine.Tests
+{
+    public class SignInTests
```

Comment:
Why do sign in tests use create user request?

Reply:
> Why do sign in tests use create user request?

To be able to login user with correct data